wiki/Beatmap_Spotlights/Seasons/2020_Autumn/zh.md

```diff
@@ -15,48 +15,48 @@ tags:
 | 阶段 | 时间 |
 | :-- | :-- |
 | 赛季开始 | 2020 年 10 月 9 日 |
-| 时移多人游戏大厅：图池 A | 2020 年 10 月 9 日 - 10 月 16 日 |
-| 时移多人游戏大厅：图池 B | 2020 年 10 月 16 日 - 10 月 23 日 |
-| 时移多人游戏大厅：图池 C | 2020 年 10 月 23 日 - 10 月 30 日 |
-| 时移多人游戏大厅：图池 D | 2020 年 10 月 30 日 - 11 月 6 日 |
+| 时移多人游戏大厅：歌单 A | 2020 年 10 月 9 日 - 10 月 16 日 |
+| 时移多人游戏大厅：歌单 B | 2020 年 10 月 16 日 - 10 月 23 日 |
+| 时移多人游戏大厅：歌单 C | 2020 年 10 月 23 日 - 10 月 30 日 |
+| 时移多人游戏大厅：歌单 D | 2020 年 10 月 30 日 - 11 月 6 日 |
 | 中场休息 | 2020 年 11 月 6 日 - 11 月 13 日 |
-| 时移多人游戏大厅：图池 A | 2020 年 11 月 13 日 - 11 月 20 日 |
+| 时移多人游戏大厅：歌单 A | 2020 年 11 月 13 日 - 11 月 20 日 |
 | 中场休息 | 2020 年 11 月 20 日 - 11 月 27 日 |
-| 时移多人游戏大厅：图池 B | 2020 年 11 月 27 日 - 12 月 4 日 |
-| 时移多人游戏大厅：图池 C | 2020 年 12 月 4 日 - 12 月 11 日 |
-| 时移多人游戏大厅：图池 D | 2020 年 12 月 11 日 - 12 月 18 日 |
+| 时移多人游戏大厅：歌单 B | 2020 年 11 月 27 日 - 12 月 4 日 |
+| 时移多人游戏大厅：歌单 C | 2020 年 12 月 4 日 - 12 月 11 日 |
+| 时移多人游戏大厅：歌单 D | 2020 年 12 月 11 日 - 12 月 18 日 |
 | 赛季结算 | 2020 年 12 月 18 日 |
 
 ### 每周多人游戏大厅
```

Comment:
Playlist D is missing on all game modes

Reply:
nice catch, might be because at the time of translation the playlist D not to be released yet, then the English were updated and the Chinese were not mark as outdated